### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by ron-huberfeld**
docs: add contribution convention section

### CODE REVIEW

**Comment by ron-huberfeld** [comment]
@yammesicka I'll move this section to a separate `CONTRIBUTING.md` file.
Do you have any preferences to the file structure?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Guide
+1. Fork the project
+2. Create a new branch (git checkout -b 'branch name')
+3. Commit your changes (git commit -m 'Add something new')
+4. Push to the branch (git push -u origin 'branch name')
+5. Open a Pull Request
+
+## Style Guide
+Follow the [commit messages specification](https://www.conventionalcommits.org/en/v1.0.0/)
```
Comment (yammesicka) [inline CONTRIBUTING.md:23]: Add dot in the end of the sentence.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+## Pull Requests
+Create feature branches.
+
+One pull request per feature - If you want to do more than one thing, send multiple pull requests.
+
+Send coherent history - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](https://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting.
+
+## Guide
+1. Fork the project
+2. Create a new branch (git checkout -b 'branch name')
```
Comment (yammesicka) [inline CONTRIBUTING.md:17]: Use ` before and after the git command.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Contributing
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (yammesicka) [inline CONTRIBUTING.md:6]: Link to it :)

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -93,3 +93,5 @@ pip install -r requirements.txt
 pip install -r dev_requirements.txt
 py.test -vvv
 ```
+### Contributing
+View [contributing guidelines](https://github.com/PythonFreeCourse/lms/blob/master/CONTRIBUING.md)
```
Comment (yammesicka) [inline README.md:97]: `.` at the end of the sentence

**Review by yammesicka — CHANGES_REQUESTED** [review]

**Pull request merged**